drop-cap/library/src/main/java/com/novoda/dropcap/DropCapView.java

```diff
@@ -91,7 +98,30 @@ private void applyCustomAttributes(Context context, AttributeSet attrs) {
         }
     }
 
-    public void setDropCapFontType(String fontPath) {
+    public void setNumberOfDropCaps(int numberOfDropCaps) {
+        this.numberOfDropCaps = numberOfDropCaps;
+
+        String text = dropCapText + copyText;
+        setText(text);
+    }
+
+    public void setText(@Nullable String text) {
+        if (enoughTextForDropCap(text)) {
+            dropCapText = String.valueOf(text.substring(0, numberOfDropCaps));
+            copyText = String.valueOf(text.subSequence(dropCapText.length(), text.length()));
```

Comment:
Any reason for not using directly `numberOfDropCaps` instead of `dropCapText.length()`?